internal/topic/topicwriterinternal/encoders_test.go

```diff
@@ -15,14 +16,32 @@ import (
 	"github.com/ydb-platform/ydb-go-sdk/v3/trace"
 )
 
+func NewTestEncoderSelector(
```

Comment:
I don't think it's necessary to create the helper function NewTestEncoderSelector. It's used only three times in the tests and only adds endpty context. I think it would be better to use EncoderSelector directly

Reply:
removed NewTestEncoderSelector